libnd4j/include/ops/BroadcastOpsTuple.h

```diff
@@ -52,6 +52,9 @@ namespace nd4j {
         static BroadcastOpsTuple Subtract();
         static BroadcastOpsTuple IGamma();
         static BroadcastOpsTuple IGammac();
+
+        static BroadcastOpsTuple Pow();
```

Comment:
tbh, this isn't a necessity... couple of "popular" methods were added by i think we're not using it actually.